docs/manual/java/concepts/Immutable.md

```diff
@@ -1,8 +1,7 @@
-# Immutable Objects
+# Using immutable objects
 
-An immutable object is an object that cannot be modified after it was created.
-
-Immutable objects have two great advantages:
+<!--- This page will move and/or change as a result of https://github.com/lagom/lagom/issues/592 -->
```

Comment:
The page will change, yes, but it must remain to explain why immutability changes everything.

The section that will change is how to achieve immutability. We'll remove support (and endorsement) for using the `immutables` library and promote `lombok` only.